src/bun.js/bindings/webcore/WebSocket.cpp

```diff
@@ -193,6 +194,7 @@ WebSocket::~WebSocket()
         break;
     }
     }
+    deref();
```

Comment:
Doesn't deref potentially call `delete` and destruct the WebSocket? is it defined behavior in C++ to call a destructor inside a destructor?

Reply:
You're right, pretty sure it's unnecessary. Tests still do not hang without it